sunpy/timeseries/sources/goes.py

```diff
@@ -175,6 +175,7 @@ def _parse_hdus(cls, hdulist):
             raise ValueError("Don't know how to parse this file")
 
         times = start_time + TimeDelta(seconds_from_start*u.second)
+        times.precision = 9
```

Comment:
Can we not set this to be for all time-series or do we just want it on some sources?

Reply:
Yeah. We should that to all the timeseries. Should I do it in this PR or in a follow up one?

Reply:
I dont see why not in this PR.